README.md

```diff
@@ -54,10 +54,9 @@ policy](https://kubernetes.io/docs/setup/release/version-skew-policy/) offered
 for Kubernetes, this project aims to support the latest three minor releases of
 Kubernetes.
 
-The default supported API is `autoscaling/v2beta2` (available since `v1.12`).
+The default supported API is `autoscaling/v2` (available since `v1.23`).
 This API MUST be available in the cluster which is the default. However for
-GKE, this requires GKE v1.15.7 according to this [GKE
-Issue](https://issuetracker.google.com/issues/135624588).
+GKE, this requires GKE v1.23+.
```

Comment:
Let's drop the sentence about GKE, that is no longer special I think. Basically drop everything from `However for GKE...`